Add Marugame Udon

peternewman opened the pull request:
I think this is the UK branding for "Marugame Seimen".

LaoshuBaby commented on data/brands/amenity/restaurant.json:
```diff
@@ -4952,6 +4952,18 @@
+        "brand:wikidata": "Q10877253",
+        "brand:wikipedia": "en:Marugame Seimen",
+        "cuisine": "noodle",
+        "name": "Marugame Udon",
```
```suggestion
        "name": "Marugame Udon"
```

peternewman commented:
Also the other one is in more places now (according to the UK site if they're the same):
https://marugame.co.uk/faqs

bhousel commented:
> Also the other one is in more places now (according to the UK site if they're the same):
https://marugame.co.uk/faqs

I added these other countries to the item in 5ce0def46, also added a `"alt_name": "Marugame Seimen"` tag, since it seems to sometimes be called that..

peternewman commented:
> I added these other countries to the item in [5ce0def](https://github.com/osmlab/name-suggestion-index/commit/5ce0def469fbbb7bd4dba91b649b64de70af7a09), also added a `"alt_name": "Marugame Seimen"` tag, since it seems to sometimes be called that..

Thanks @bhousel , I can only guess that they changed the second part of the name due to unfortunately sounding similar to bodily fluids that wouldn't be very appetising... 🤮
